Examples/Python/python/acts/examples/__init__.py

```diff
@@ -358,7 +358,7 @@ def __init__(self, *args, **kwargs):
                     n.append(self.FpeMask(loc, t, count))
                 kwargs["fpeMasks"] = n
 
-            kwargs["fpeMasks"] = kwargs.get("fpeMasks", []) + self._getAutoFpeMasks()
+        kwargs["fpeMasks"] = kwargs.get("fpeMasks", []) + self._getAutoFpeMasks()
```

Comment:
I guess this comes from a different branch?

Reply:
Right, I've moved this here and there because I was having issues.